frontend/components/CourseListings/CoordinatorList.tsx

```diff
@@ -5,7 +5,7 @@ import CardContent from '@mui/material/CardContent';
 import Typography from '@mui/material/Typography';
 import Button from '@mui/material/Button';
 import Stack from '@mui/material/Stack';
-import Link from '@mui/material/Link';
```

Comment:
this is a very powerful change @chayG5 . This is based on https://nextjs.org/docs/api-reference/next/link

Essentially it makes the switching of the page extremely fast. Try it out for yourself!

Reply:
Here's the explanation... NextJS can render both server-side and client-side. If we render in "server-side" then the server will process it and deliver the files via the internet. In client-side, the browser processes the changes... Remember that most often, CPUs are faster than waiting for network requests.

Next Link allows us to do client-side rendering when switching pages, while MUI link is generic for just changing a link. In other words, "Next Link" is like "anchor tags `<a>` with steroids` while MUI link is just a an anchor tag with wrapper of the MUI tags